### PR TITLE
Add ability to stress virtual machine scale set instances

### PR DESCRIPTION
**Pull request by maximmold**
Signed-off-by: Maxim Moldenhauer <moldenhauer_maxim@bah.com>

### CODE REVIEW

**Comment by maximmold** [comment]
Clean version of https://github.com/chaostoolkit-incubator/chaostoolkit-azure/pull/81

File: chaosazure/vmss/actions.py [hunk 1]
```diff
@@ -7,6 +8,7 @@
 from chaosazure import init_client
 from chaosazure.rgraph.resource_graph import fetch_resources
 from chaosazure.vmss.constants import RES_TYPE_VMSS
+from chaosazure.machine.constants import OS_LINUX, OS_WINDOWS
```
Comment (Lawouach) [inline chaosazure/vmss/actions.py]: This should be ordered alphabetically. We probably could add [isort](https://pypi.org/project/isort/) to the package one day.

**Comment by maximmold** [comment]
I moved the scripts out from under the machine directory since they were being used from the vmss package as well

**Comment by Lawouach** [comment]
This makes total sense.

Okay, thanks. Will merge this one now.

**Pull request merged**